packages/extension/package.json

```
@@ -3,68 +3,56 @@
-    "@emotion/react": "^11.7.1",
-    "@emotion/styled": "^11.6.0",
-    "@onflow/fcl": "^1.0.3-alpha.0",
+    "@chakra-ui/icons": "^2.2.4",
+    "@chakra-ui/layout": "^2.3.1",
+    "@chakra-ui/react": "^2.10.9",
+    "@chakra-ui/system": "^2.6.2",
+    "@chakra-ui/toast": "^2.1.0",
+    "@emotion/react": "^11.14.0",
+    "@emotion/styled": "^11.14.1",
+    "@noble/hashes": "^1.8.0",
+    "@onflow/fcl": "1.20.0",
+    "@onflow/rlp": "^1.2.3",
+    "@onflow/transport-http": "1.13.0",
+    "@onflow/types": "^1.4.1",
     "browser-passworder": "^2.0.3",
     "buffer": "^6.0.3",
-    "elliptic": "^6.5.4",
-    "framer-motion": "^4.1.17",
+    "elliptic": "^6.6.1",
+    "framer-motion": "^12.23.9",
     "https-browserify": "^1.0.0",
-    "node-forge": "^1.2.1",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0",
-    "react-icons": "^4.3.1",
-    "react-router": "^5.2.1",
-    "react-router-dom": "^5.3.0",
+    "node-forge": "^1.3.1",
+    "react": "^19.1.0",
+    "react-dom": "^19.1.0",
+    "react-icons": "^5.5.0",
+    "react-router": "^5.3.4",
+    "react-router-dom": "^5.3.4",
     "react-router-transition": "^2.1.0",
     "sha3": "^2.1.4",
     "stream-http": "^3.2.0",
-    "url": "^0.11.0"
+    "url": "^0.11.4"
   },
   "devDependencies": {
-    "@babel/core": "^7.17.0",
-    "@babel/plugin-proposal-class-properties": "^7.16.7",
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-react": "^7.16.7",
-    "@types/chrome": "^0.0.177",
-    "@types/react": "^17.0.39",
-    "@types/react-dom": "^17.0.11",
-    "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.2.3",
-    "babel-preset-react-app": "^10.0.1",
-    "clean-webpack-plugin": "^4.0.0",
-    "copy-webpack-plugin": "^7.0.0",
-    "css-loader": "^6.6.0",
-    "eslint": "^8.8.0",
-    "eslint-config-react-app": "^7.0.0",
+    "@types/chrome": "^0.1.1",
+    "@types/react": "^19.1.8",
+    "@types/react-dom": "^19.1.6",
+    "@vitejs/plugin-react": "^4.7.0",
+    "eslint": "^9.31.0",
+    "eslint-config-react-app": "^7.0.1",
     "eslint-plugin-flowtype": "^8.0.3",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-jsx-a11y": "^6.5.1",
-    "eslint-plugin-react": "^7.28.0",
-    "eslint-plugin-react-hooks": "^4.3.0",
-    "file-loader": "^6.2.0",
-    "html-loader": "^3.1.0",
-    "html-webpack-plugin": "^5.5.0",
-    "node-polyfill-webpack-plugin": "^1.1.4",
-    "node-sass": "^6.0.1",
-    "prettier": "^2.5.1",
-    "sass-loader": "^12.4.0",
-    "source-map-loader": "^3.0.1",
-    "style-loader": "^3.3.1",
-    "terser-webpack-plugin": "^5.3.1",
-    "ts-loader": "^9.2.6",
-    "typescript": "^4.5.5",
-    "webpack": "^5.68.0",
-    "webpack-cli": "^4.9.2"
+    "eslint-plugin-import": "^2.32.0",
+    "eslint-plugin-jsx-a11y": "^6.10.2",
+    "eslint-plugin-react": "^7.37.5",
+    "eslint-plugin-react-hooks": "^5.2.0",
+    "prettier": "^3.6.2",
+    "sass": "^1.89.2",
+    "typescript": "^5.8.3",
+    "vite": "^7.0.6",
+    "vite-plugin-node-polyfills": "^0.24.0"
   }
```

```suggestion
  },
  "type": "module"
```

Needed this to be able to run `pnpm run build`